Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Create image for the application
+FROM maven:3.8.1-openjdk-17 AS build
+
+#Set the working directory
+WORKDIR /app
+
+#Copy pom.xml to current working directory
+COPY pom.xml .
+RUN mvn dependency:go-offline -B 

```

Comment:
Please add a comment, mentioning what go-offline -B would do.
Is there any benefit since we are already packaging all of the dependencies on line 13 ?